SEARCH.py

```diff
@@ -1,7 +1,6 @@
 import random
 from timeit import default_timer as timer
-#changes made in new_branch
-#change in demo branch
+#changes made in new branch
 

```

Comment:
any python program, start with a main function 

something like this

# start.py

import sys

from namely import print_user_name

# my_name = "Dinsdale"

def main():
    try:
        print_user_name()
    except ValueError as ve:
        return str(ve)

if __name__ == "__main__":
    main()